src/content/Colors/Colors.tsx

```diff
@@ -25,62 +25,22 @@ interface IColorsProps {
 }
 
 interface IColorGroups {
-  brand: Array<keyof IBrandColors>;
-  actions: Array<keyof IActionColors>;
-  neutral: Array<keyof INeutralColors>;
-  notifications: Array<keyof INotificationColors>;
+  brand: Array<string>;
+  actions: Array<string>;
+  neutral: Array<string>;
+  notifications: Array<string>;
```

Comment:
Note that this is typed according to what`Object.keys` below returns.

Given this file is just for documentation I think is a fair trade off 🙄